terms_of_service.md

```diff
@@ -1,5 +1,189 @@
+
+0.3. To transmit, or procure the sending of, any advertising or promotional material, including any “junk mail”, “chain letter,” “spam,” or any other similar solicitation.
+
+0.4. To impersonate or attempt to impersonate Company, a Company employee, another user, or any other person or entity.
+
+0.5. In any way that infringes upon the rights of others, or in any way is illegal, threatening, fraudulent, or harmful, or in connection with any unlawful, illegal, fraudulent, or harmful purpose or activity.
+
+0.6. To engage in any other conduct that restricts or inhibits anyone’s use or enjoyment of Service, or which, as determined by us, may harm or offend Company or users of Service or expose them to liability.
+
+Additionally, you agree not to:
+
+0.1. Use Service in any manner that could disable, overburden, damage, or impair Service or interfere with any other party’s use of Service, including their ability to engage in real time activities through Service.
+
+0.2. Use any robot, spider, or other automatic device, process, or means to access Service for any purpose, including monitoring or copying any of the material on Service.
+
+0.3. Use any manual process to monitor or copy any of the material on Service or for any other unauthorized purpose without our prior written consent.
+
+0.4. Use any device, software, or routine that interferes with the proper working of Service.
+
+0.5. Introduce any viruses, trojan horses, worms, logic bombs, or other material which is malicious or technologically harmful.
+
+0.6. Attempt to gain unauthorized access to, interfere with, damage, or disrupt any parts of Service, the server on which Service is stored, or any server, computer, or database connected to Service.
+
+0.7. Attack Service via a denial-of-service attack or a distributed denial-of-service attack.
+
+0.8. Take any action that may damage or falsify Company rating.
+
+0.9. Otherwise attempt to interfere with the proper working of Service.
+
+6. Analytics
+
+We may use third-party Service Providers to monitor and analyze the use of our Service.
+
+7. No Use By Minors
+
+Service is intended only for access and use by individuals at least eighteen (18) years old. By accessing or using Service, you warrant and represent that you are at least eighteen (18) years of age and with the full authority, right, and capacity to enter into this agreement and abide by all of the terms and conditions of Terms. If you are not at least eighteen (18) years old, you are prohibited from both the access and usage of Service.
+
+8. Accounts
+
+When you create an account with us, you guarantee that you are above the age of 18, and that the information you provide us is accurate, complete, and current at all times. Inaccurate, incomplete, or obsolete information may result in the immediate termination of your account on Service.
```

Comment:
```suggestion
8. Accounts

```
remove underage gatekeeping clause

Reply:
If you are going to insist on that, the way to do it would be to set the limit at 13 and ban all nsfw material and conversation, including from remote instances. You can't just remove it as if it was here for nothing.